robot_smach_states/src/robot_smach_states/manipulation/grasp_point_determination.py

```diff
@@ -34,6 +34,26 @@ def get_grasp_pose(self, entity, arm):
         :param arm: arm to use
         :return: FrameStamped with grasp pose in map frame
         """
+        try:
+            # Try to compute the grasp pose based on the entity
+            return self._get_grasp_pose(entity, arm)
+        # noinspection PyBroadException
+        except Exception as e:
+            rospy.logwarn("Cannot compute grasp pose ({}), defaulting to entity position and "
+                          "orientation the same as the robot".format(e.message))
```

Comment:
Actually I am not sure about this approach. We are essentially saying. "If we don't have a shape we will just grab at the coordinates" I can imagine this will at best fail to grasp the object and at worse jam the hand into a cabinet. I know this is needed for your test but I don't see an application in normal conditions.

Reply:
Reverted in 575f91f6509e923e85a9e5147861a310f545ce8e and patched the method from the testscript in 09de5169d321815ae17541f8b80f5dd0331ad6cd . Is this more to your liking?

Reply:
It is, though I wonder why you did not just add a shape to your test_entity? since we are testing the grab state I would be hesitant to overwrite functions from it.

Reply:
After trying to manually add a shape to an entity myself I understand the decision.